samples/SampleMvvmApp/Views/DashboardView.xaml

```diff
@@ -46,13 +46,18 @@
                ScrollViewer.VerticalScrollBarVisibility="Visible"
                SelectedIndex="{Binding ListItemSelected}"
                SelectionMode="Single">
-        <ListBox.DataTemplates>
-          <DataTemplate>
+        <ListBox.ItemTemplate>
+          <!--
+          OLD: <DataTemplate>
+          NEW: <DataTemplate DataType="{x:Type system:String}">
+          https://github.com/AvaloniaUI/Avalonia/pull/8221
+          -->
+            <DataTemplate DataType="{x:Type system:String}">
             <TextBlock Text="{Binding .}"
                        FontSize="10"
                        TextWrapping="NoWrap" />
           </DataTemplate>
-        </ListBox.DataTemplates>
+        </ListBox.ItemTemplate>
```

Comment:
Note, you don't necessary need DataType, if you use ListBox.ItemTemplate instead ListBox.DataTemplates.
It won't cause any exceptions, as ListBox.ItemTemplate is way more specific and won't pollute children with the template.
Although, it's always a good idea to setup more strict types. Especially with x:CompileBindings.